src/pcp/js/settingsPanel.js

```diff
@@ -0,0 +1,472 @@
+"use strict";
+(function () {
+    var fluid = window.fluid,
+        gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.pcp");
+
+
+    /**
+     * Creates the binding with the already rendered DOM elements.
+     * Expects: widget configuration and model
+     */
+    fluid.defaults("gpii.pcp.settingPresenter", {
```

Comment:
Now this rendering system has stabilised somewhat, it should be given some in-browser unit tests - consult https://github.com/GPII/universal/tree/master/gpii/node_modules/lifecycleManager/test for our basic pattern for these